### PR TITLE
Bound cloud storage types on PVS prov. to API returned values

### PR DESCRIPTION
**Pull request by iv1111**

### CODE REVIEW

File: app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb [hunk 1]
```diff
@@ -24,8 +24,11 @@ def allowed_sys_type(_options = {})
   end
 
   def allowed_storage_type(_options = {})
-    # TODO: replace with api provided values, once issue '115' is solved and merged
-    {0 => "None", 1 => "Tier 1", 2 => "Tier 3"}
+    ar_ems = ar_ems_get
+    ar_storage_types = ar_ems&.cloud_volume_types
+    storage_types = ar_storage_types&.map&.with_index { |storage_type, i| [i + 1, storage_type['name']] }
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb:29]: Similar comment as #57

**Review by agrare — APPROVED** [review]

File: app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb (hunk 1 above)
Comment (iv1111) [reply app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/provision_workflow.rb]: Done. See the diff in "Files".

**Comment by iv1111** [comment]
Done the changes btu let's wait for Travis to finish.

**Comment by agrare** [comment]
Oops accidentally reopened thinking travis was red not clodeclimate

**Pull request merged**